core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/ldap/ReadOnlyLDAPUserStoreManager.java

```diff
@@ -80,6 +80,8 @@
 
     public static final String MEMBER_UID = "memberUid";
     private static final String OBJECT_GUID = "objectGUID";
+    protected static final String ROLE_USER_LIST_RANGE = "RoleUserListRange";
```

Comment:
Shall we rename the property to be "MembershipAttributeRange" since it is more meaningful and we already have "MembershipAttribute" property.